lib/bolt/cli.rb

```diff
@@ -6,6 +6,8 @@
 require 'io/console'
 require 'logging'
 require 'optparse'
+require 'securerandom'
```

Comment:
Why did you bring in securerandom?

Reply:
Whoops, was generating the uuid here in a prior iteration. Will remove this.